packages/dev/core/src/Meshes/csg2.ts

```diff
@@ -0,0 +1,513 @@
+import { Mesh } from "./mesh";
+import type { IDisposable, Scene } from "core/scene";
+import { VertexData } from "./mesh.vertexData";
+import { VertexBuffer } from "./buffer";

```

Comment:
```suggestion
import { VertexBuffer } from "../Buffers/buffer";
```
because at some point, I think we will remove the `Meshes/buffer.ts` file.